tests/unit/amo/components/TestAddonAuthorLinks.js

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+
+import AddonAuthorLinks, {
+  AddonAuthorLinksBase,
+} from 'amo/components/AddonAuthorLinks';
+import { createInternalAddon } from 'core/reducers/addons';
+import {
+  dispatchClientMetadata,
+  dispatchSignInActions,
+  fakeAddon,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  let store;
+  const slug = 'some-slug';
+
+  beforeEach(() => {
+    store = dispatchClientMetadata().store;
+  });
+
+  function render(props = {}) {
+    return shallowUntilTarget(
+      <AddonAuthorLinks
+        addon={props.addon || createInternalAddon(fakeAddon)}
+        i18n={fakeI18n()}
+        store={store}
+        {...props}
+      />,
+      AddonAuthorLinksBase,
+    );
+  }
+
+  const renderWithPermissions = ({
+    addon = createInternalAddon({
+      ...fakeAddon,
+      slug,
+      authors: [
+        {
+          id: 10642015,
+          name: 'avgupt',
+          picture_url: 'http://cdn.a.m.o/myphoto.jpg',
+          url: 'http://a.m.o/en-GB/firefox/user/avgupt/',
+          username: 'avgupt',
+        },
+      ],
+    }),
+    userId,
+  }) => {
+    dispatchSignInActions({ store, userId });
+    return render({ addon });
+  };
+
+  it('returns nothing if the userId is null / not logged in', () => {
+    const root = renderWithPermissions({});
```

Comment:
Shouldn't we pass `userId: null`?

Reply:
I thought it's the same thing, I'll change it.

Reply:
`userId: null` doesn't work.
![Screenshot from 2020-03-25 13-46-13](https://user-images.githubusercontent.com/58681517/77515761-25210d80-6e9f-11ea-9b88-3a4f73688430.png)

Reply:
ha, I see. I'll offer an alternative then